samples/testing/dump_operation/test_dump_operation.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import qsharp
+from qsharp.utils import dump_operation
+
+def test_empty_operation() -> None:
+    qsharp.init(target_profile=qsharp.TargetProfile.Unrestricted)
```

Comment:
Do we need to set this profile explicitly every time? I think it's the default setting, my tests get away with not using it, just saying qsharp.init()